src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -159,6 +162,18 @@ public string Format()
                 { nameof(HttpAutoCollectionOptions.EnableResponseHeaderInjection), HttpAutoCollectionOptions.EnableResponseHeaderInjection }
             };
 
+            JObject dependencyTrackingOptions = new JObject
```

Comment:
do we need to check if `DependencyTrackingOptions` is null here?

Reply:
fixed